packages/ui/form/PhoneInput.tsx

```diff
@@ -30,7 +30,9 @@ function BasePhoneInput({ name, className = "", onChange, ...rest }: PhoneInputP
         required: rest.required,
         placeholder: rest.placeholder,
       }}
-      onChange={(value) => onChange(value)}
+      onChange={(value) => {
+        onChange("+" + value);
```

Comment:
I noticed that new react-phone-input-2 doesn't have + in the value it gives which is required by isValidPhoneNumber